bower.json

```diff
@@ -37,7 +37,7 @@
     "angular-schema-form-bootstrap": "^0.2.0",
     "angular-filter": "^0.5.16",
     "ng-file-upload": "^12.2.13",
-    "angucomplete-alt": "^1.3.1",
+    "angucomplete-alt": "git://github.com/fgourichon/angucomplete-alt.git#1.3.2" ,
```

Comment:
@ouranos Should we move this fork of `angucomplete-alt` on a maestrano repo?

Reply:
Yup

Reply:
Pointed to maestrano/angucomplete-alt, I'll let you confirm when the report is available, I don't have the rights to create it.